deploy/kubernetes/overlays/linux/base/noderegistrar.yaml

```diff
@@ -0,0 +1,17 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-node
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-driver-registrar
+          args:
+            - "--v=5"
+            - "--csi-address=/csi/csi.sock"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock"
+          lifecycle:
```

Comment:
You can actually remove the entire prestop hook, it doesn't work anymore and the solution is not going to use this: https://github.com/kubernetes-csi/node-driver-registrar/issues/21

Reply:
Does the same need to be done for the windows registrar?